src/index.ejs

```diff
@@ -60,7 +60,7 @@
           useModal: useModal,
           buttonId: 'onfido-button',
           onReady: function() {
```

Comment:
Oh yeah I see. Why bring it back? In general I think the more we remove from the public API the better.